edn/src/types.rs

```diff
@@ -214,12 +264,6 @@ impl Value {
     }
 }
 
-impl From<f64> for Value {
```

Comment:
It's a leftover from the previous implementation, but makes this a bit more symmetrical/generic so I'll keep it.